examples/examples.ipynb

```diff
@@ -165,6 +165,38 @@
     "\n",
     "solve"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "$$ \\displaystyle \\frac{-b + \\sqrt{b^{{2}} - {4}ac}}{{2}a} $$"
+      ],
+      "text/plain": [
+       "<latexify.frontend.LatexifiedFunction at 0x7f58824de860>"
+      ],
+      "text/plain": [
+       "The above equation is sent to a new file, latexifyExample.tex"
+      ]
```

Comment:
It looks this is not a real output. I think it is better to put this paragraph as a comment of the code block.

Reply:
Good point. Perhaps rather than relocate this, removal may be sufficient if the current comment at the top of the block is descriptive enough.